mowgli_etl/pipeline/wdc/wdc_dimension_parser.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC, abstractmethod
+
+
+from mowgli_etl.pipeline.wdc.wdc_product_dimensions import WdcProductDimensions
+
+class WdcDimensionParser(ABC):
+    @abstractmethod
+    def parse(self,*,information:dict) -> WdcProductDimensions:
```

Comment:
You'll need to use a proper type here, `typing.Dict`. But this is fine for now.

Reply:
Or even better, create a NamedTuple/@dataclass that exactly models what's coming out of the WDC file. Much better than passing around a dict.